docs/guides/social-signin/github.mdx

```diff
@@ -67,15 +67,15 @@ makes a request to
 and adds the user data to `std.extVar('claims')`. Learn what data is available
 by reading the
 [GitHub Scopes documentation](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/).
-However, not all GitHub scopes are supported. Check the list of supported fields
+Not all GitHub scopes are supported. Check the list of supported scopes in
 [the source code](https://github.com/ory/kratos/blob/v0.2.1-alpha.1/selfservice/strategy/oidc/provider_github.go#L72-L98).
 
 :::
 
 In this sample Jsonnet snippet, `email_primary` is mapped to the identity
 schema's `traits.email`:
 
-```github.jsonnet
+```json 
```

Comment:
@piotrmsc GitHub uses the `user:email` scope. Considering that, the jsonnet mapping doesn't seem to be correct. Can you please check that? If the jsonnet is wrong, could you let me know how to fix it? Thanks :)